tips.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: default
+title: Tips for Working with AI Jr Dev
+---
+
+# {{ page.title }}
+
+Working effectively with an AI pair programmer like AI Jr Dev involves clear communication and providing the right context. Here are some tips:
+
+## Provide Necessary Context
+
+*   **Include Relevant Files:** When requesting code changes, always ensure the issue or prompt includes the full contents of all files that need to be modified or are directly related to the change. This is the most crucial piece of context.
```

Comment:
You're actually pretty good at finding contents of related files, at times.

You should rephrase this. Something similar to, "Mention files that may need to be written or modified. I'm a lot better at my job when I know where to look"